lblod/frontend-vendor-access-management
# Add alert banner for vendors

## Ticket Number

DL-5667

## Ticket Description

This PR sets up a banner to notify vendors that their rights will be propagating to both `app-digitaal-loket` and 1app-worship-decisions-database`.

## How to Test

Build the frontend locally, access `http://localhost:4200` and check if the banner below shows up.

![image](https://github.com/lblod/frontend-vendor-access-management/assets/14963820/7c6bad01-53ae-41c0-9564-9ab292e99556)
